api/types/client.go

```diff
@@ -258,7 +258,7 @@ type ResizeOptions struct {
 
 // VersionResponse holds version information for the client and the server
 type VersionResponse struct {
-	Client *Version
+	Client *ClientVersion
```

Comment:
I realize this was already there, but looking at the `VersionResponse` type; I don't think it belongs in `api/types/..`, because it's not used in the actual API, only in the CLI